### DISCUSSION TITLE
Content creation

### DISCUSSION THREAD

**Discussion by Homap** [WeAreMahsaAmini/Core#176] [Introductions]
Salam,

My name is Homa. I am a researcher in Sweden. I can help with content creation, I am good at writing in English, French and Swedish and can translate from Farsi to any of these languages. I can also code in Bash and Python if that is helpful. Please assign me any task you think I can be useful in and I do my best.

Mersi az kare alitoon,
Homa

**Comment by fatihany98** [comment]
Hi,
My name is Hany.

I am a master's student in Germany. I can help with content creation and can translate from Farsi to Deutsch. I can also gather different photos that have during these protests. Also, I already designed some websites with CMS like WordPress. Please assign me any task you think I can be useful in and I do my best.

Thanks a lot for your valuable project,
Hany

**Comment by sepehrbg** [comment]
Hi Eveyone,
My name is Sepehr. I am .Net developer at Sweden but for now I can start contributing by translation from Persian to Turkish. Please assign me any task regarding translation or back-end development.
